packages/googleanalytics/events/gtagAddToCart/GtagAddToCart.graphql

```diff
@@ -1,5 +1,11 @@
 fragment GtagAddToCart on Cart @inject(into: ["CartItemCountChanged"]) {
   items {
+    product{
+      manufacturer
```

Comment:
`manufacturer` isn't a system attribute, so isn't allowed to be used.

Reply:
Do you know a way to get the brand of a product as a system attribute?

Reply:
Brand isn't a system attribute unfortunately :(

All values are prefixed with `item_`, maybe allow users to inject something like this that the user can add locally?

```
fragment MyAttr on CartItem @inject(into: ["CartItemCountChanged"]) {
   product {
      item_brand: manufacturere
   }
}
```